### DISCUSSION TITLE
Feature Request: Add caniuse link to Tailwind CSS documentation

### DISCUSSION THREAD

**Discussion by adarsh4d** [tailwindlabs/tailwindcss#7644] [Ideas]
This is not related to tailwind at all. But it would be helpful if a link to caniuse page for each class is present in the documentation. So we can know the browser support and tailwind documentation will be one stop solution for all. 

If this doesn't align with tailwind, please feel free to close this.

**Comment by reinink** [comment]
Hey thanks for sharing this idea! This honestly would be nice to have some day, but it's probably a bigger project than it sounds, so it isn't something we've been able to get to yet.

I've moved this over to our discussions, as this is a better place for ideas like this, since it's not really an issue.
